lib/matplotlib/figure.py

```diff
@@ -884,20 +884,20 @@ def add_axes(self, *args, **kwargs):
         rect : sequence of float
             A 4-length sequence of [left, bottom, width, height] quantities.
 
-        projection :
-            ['aitoff' | 'hammer' | 'lambert' | 'mollweide' | \
-'polar' | 'rectilinear'], optional
+        projection : {'aitoff', 'hammer', 'lambert', 'mollweide', \
+        'polar', 'rectilinear'}, optional
```

Comment:
Please leave as it was, the new form looks ugly at the terminal / pydoc.

Reply:
How about `projection : string` and then specifying the list of allowed strings underneath?

Reply:
Sounds like a better idea I think.

Reply:
Specifying the list of allowed strings underneath is reasonable for long lists. I'm still not sure how you want it. Does *ugly* refer to changed style `[ -> {` or the indentation. Reverted for now.